tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -880,20 +891,37 @@ <h2>Create a distance feature</h2>
       <div class="main-vertical">
         <div class="main-bottom-bar">
           <div class="datapoint-left-controls-holder">
+            <div class="button-prefix-label">Partial Dependence Plots:</div>
+            <div>
+              <paper-button class="control-button" disabled$="[[!areExamplesEditable_(modelName, inferenceAddress)]]"
+                  on-click="showGlobalPartialDependencePlots_" alt="show global partial dependence plots">
```

Comment:
consider using `on-tap` which works also with touch devices :)

Reply:
didn't realize this. did a global search and replace